plugins/community/neoeinstein-prost-crate/v0.3.1/Dockerfile

```
@@ -0,0 +1,12 @@
+# syntax=docker/dockerfile:1.4
+FROM rust:1.68.0-alpine3.17 as builder
```

```suggestion
FROM rust:1.70.0-alpine3.18 as builder
```

When introducing new plugin versions, we usually take it as an opportunity to bump to the latest Docker images.